core/templates/components/button-directives/social-buttons.component.ts

```diff
@@ -17,9 +17,6 @@
  */
 
 import { Component, OnInit } from '@angular/core';
-
-import { downgradeComponent } from '@angular/upgrade/static';
-
```

Comment:
This line should not be removed. There should be a line break between angular imports and oppia imports.

Reply:
Done.